pkg/sidecar/identity/member.go

```diff
@@ -73,6 +85,16 @@ func (m *Member) GetSeeds(ctx context.Context, coreClient v1.CoreV1Interface, ex
 
 	if len(otherPods) == 0 {
 		// We are the only one, assuming first bootstrap.
+
+		podOrdinal, err := naming.IndexFromName(m.Name)
+		if err != nil {
+			return nil, fmt.Errorf("can't get pod index from name: %w", err)
+		}
+
+		if m.RackOrdinal != 0 || podOrdinal != 0 {
```

Comment:
sounds like rack ordinal could be shortcut it at the start before even getting the pod ordinal